app/lib/service/download_counts/sync_download_counts.dart

```diff
@@ -56,120 +56,157 @@ Map<String, int> _extractDayCounts(Map<String, dynamic> json) {
   return dayCounts;
 }
 
-Future<bool> processDownloadCounts(
-    String downloadCountsFileName, DateTime date) async {
-  final downloadCountsBucket =
+Future<Set<String>> processDownloadCounts(DateTime date) async {
+  final downloadCountsFileName = [
```

Comment:
Is this still used (except for reporting where we probably should use the actual problematic file name)?

Reply:
nope, removed it.